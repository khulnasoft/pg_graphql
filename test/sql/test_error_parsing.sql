select graphql.resolve(
    'query MyQuery { { extra brack }'
)
