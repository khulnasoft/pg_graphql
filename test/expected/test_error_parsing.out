select graphql.resolve(
    'query MyQuery { { extra brack }'
)
                                                      resolve                                                       
--------------------------------------------------------------------------------------------------------------------
 {"errors": [{"message": "query parse error: Parse error at 1:17\nUnexpected `{[Punctuator]`\nExpected `Name`\n"}]}
(1 row)

